src/components/blocks/Rte/Rte.stories.ts

```diff
@@ -29,7 +29,8 @@ export default meta;
 
 export const RteStory: StrictStory = {
   args: {
-    // blockType: 'textBlock',
-    text: sampleRte3,
+    colorMode: 'white',
```

Comment:
the color mode change does not work in storybook (for the font), and of course this way it can not work with the background.

i'd recommend to create 3 stories, 1 for each color mode. that way we get a clean automatic visual test for each variant.